ext/devcontainer/src/azd/devcontainer-feature.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "azd",
+    "version": "1.0.0",
+    "name": "Azure Developer CLI",
+    "documentationURL": "https://github.com/azure/azure-dev/tree/main/ext/devcontainer/src/azd",
+    "description": "Installs the Azure Developer CLI along with needed dependencies.",
+    "options": {
+      "version": {
+        "type": "string",
+        "proposals": [
+          "latest"
+        ],
+        "default": "latest",
+        "description": "Select or enter an Azure Developer CLI version. (Available versions may vary by Linux distribution.)"
+      }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+              "ms-azuretools.vscode-bicep",
+              "ms-vscode.vscode-node-azure-pack",
```

Comment:
Should we add this extension? I think it makes sense, since you're going to be working with Azure, or does it get pulled in via `ms-azuretools.azure-dev`?